packages/cli/README.md

```
@@ -0,0 +1,310 @@
+## Commands
+
+### `init [directory]`
+Initialize a new SpecSafe project.
+
+```bash
+specsafe init my-project
+cd my-project
+```
+
+Creates the following structure:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)